app/Filament/Widgets/OldStartScan.php

```diff
@@ -15,20 +16,46 @@ class StartScan extends Widget
 
     public function start()
     {
-        // Make an entry in scan table
-        $scan = new Scan;
+        $validator = Validator::make([
+            'username' => $this->username,
+            'token' => $this->token,
+        ], [
+            'username' => 'required',
+            'token' => 'required',
+        ]);
```

Comment:
better to use something like

```
        $validated = $this->validate([ 
            'username' => 'required',
            'token' => 'required',
        ]);
 
        Scan::create($validated);
```

Reply:
Yes, but I think this is the more straightforward approach.